4_report_maker.R

```diff
@@ -42,15 +39,11 @@ portfolio_overview$portfolio_name <- clean_punctuation(portfolio_overview$portfo
 report_list <- get_report_list(portfolio_overview)
 
 
-#template <- read_utf8_tex(paste0(getwd(),"/Templates/",templateversion,".tex"))
+template <- readLines(paste0(dirname("/Templates/",templateversion,".tex"),encoding = "UTF-8"))
```

Comment:
this crashes the report for me, not sure why and what it's for

Reply:
the `read_utf8_text()` line or the `readLines`?
This functin loads up the "default" template tex file that is used in report generation. The biggest hurdle we had with getting report printing to work on multiple computers was file encoding, so we forced the encoding to be UTF-8 in hopes that that would fix things.

Reply:
the readLines does not work.. one issue seems to be that dirnames expects one input, which can be fixed by wrapping the three inputs in another paste0(), but even then, I won't find the corresponding directory